Makefile

```
@@ -38,9 +38,29 @@ install: build
 
+# Acceptance tests against a REAL Control API. Requires ABLY_ACCOUNT_TOKEN (and
+# optionally ABLY_URL, e.g. staging). Setting TF_ACC makes TestMain stand aside
+# so the suite hits the real API instead of the fake.
 testacc:
 	TF_ACC=1 go test $(TEST) -v $(TESTARGS) -timeout 120m
+
+# Regenerate Terraform schema/model code from the vendored Control API spec
+# (codegen/swagger.yaml) using HashiCorp's tech-preview codegen tools. The
+# generated code lands in internal/provider/codegen/. See codegen/README.md for
+# how to refresh the spec and the current scope/caveats.
+generate:
```

checkmake (0.3.2)

[warning] Target body for "generate" exceeds allowed length of 5 lines (8).

(maxbodylength)